tests/main_test.py

```diff
@@ -151,23 +156,23 @@ def test_group_no_desc(self):
         data = {
             'name': 'test group name',
         }
-        content, status_code = self.do_post('/group/create', {'data': data})
-        self.group = content['data']
-        assert status_code == 200
+        response = self.do_post('/group/create', {'data': data})
+        self.group = response.json()['data']
+        self.ensure_status_code_msg(response, 200, "Group successfully created.")
 
         # get the group
-        content, status_code = self.do_post('/group/info', {'id': self.group['_id']['$oid']})
-        assert status_code == 200
-        assert content['data']['name'] == 'test group name'
+        response = self.do_post('/group/info', {'id': self.group['_id']['$oid']})
+        self.ensure_status_code_msg(response, 200, "Group successfully retrieved.")
+        assert response.json()['data']['name'] == 'test group name'
 
         # delete the group
-        content, status_code = self.do_post('/group/delete', {'id': self.group['_id']['$oid']})
-        assert status_code == 200
+        response = self.do_post('/group/delete', {'id': self.group['_id']['$oid']})
+        self.ensure_status_code_msg(response, 200, "Group successfully deleted.")
```

Comment:
@tdbucks after this you may also want to add another check that calls /group/info